dm/dm-open-api.md

```diff
@@ -392,14 +392,14 @@ curl -X 'PATCH' \
 
 ### 请求 URI
 
- `PATCH /api/v1/sources/{source-name}/stop-relay`
+ `POST /api/v1/sources/{source-name}/stop-relay`
```

Comment:
https://github.com/pingcap/tiflow/blob/3ab3c7d2248193ae356a00a7a720f4eecf7ad709/dm/dm/master/openapi.go#L277

can you provide a reference file that review can check the description?

Reply:
thanks, the source-truth is in [spec yaml](https://github.com/pingcap/tiflow/blob/17e48b2c64b9d3dfc0cbd210c3ca0e882c437abb/dm/openapi/spec/dm.yaml#L180), the above comment was outdated, i will fix those comments in other pr